<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AnalysisWidget</class>
 <widget class="QWidget" name="AnalysisWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>446</width>
    <height>566</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>150</width>
    <height>80</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Analysis</string>
  </property>
  <layout class="QVBoxLayout">
   <property name="spacing">
    <number>3</number>
   </property>
   <property name="leftMargin">
    <number>3</number>
   </property>
   <property name="topMargin">
    <number>3</number>
   </property>
   <property name="rightMargin">
    <number>3</number>
   </property>
   <property name="bottomMargin">
    <number>3</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="topControls">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>&amp; MPv:</string>
       </property>
       <property name="buddy">
        <cstring>vpcount</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="vpcount">
       <property name="toolTip">
        <string>Number of Multi PV </string>
       </property>
       <property name="minimum">
        <number>2</number>
       </property>
       <property name="maximum">
        <number>9</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="engineList">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>选择引擎</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="bookList">
       <property name="toolTip">
        <string>开局库选择</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="analyzeButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>使用</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTextBrowser" name="variationText">
     <property name="openLinks">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>vpcount</tabstop>
  <tabstop>engineList</tabstop>
  <tabstop>analyzeButton</tabstop>
  <tabstop>variationText</tabstop>
 </tabstops>
 <resources>
  <include location="../res/chessboard/chessboard.qrc"/>
 </resources>
 <connections/>
</ui>
